ctapipe/image/showermodel.py

```diff
@@ -0,0 +1,102 @@
+import astropy.units as u
+import numpy as np
+from scipy.stats import multivariate_normal
+from scipy.spatial.transform import Rotation as R
+
+
+__all__ = [
+    "Gaussian",
+]
+
+
+class Gaussian:
+    @u.quantity_input(
+        x=u.m,
+        y=u.m,
+        phi=u.deg,
+        theta=u.deg,
+        first_interaction=u.m,
+        width=u.m,
+        length=u.m,
+    )
+    def __init__(self, n_ch, x, y, phi, theta, first_interaction, width, length):
+        """Create a 3D gaussian shower model for imaging.
+
+        Parameters
+        ----------
+        n_ch : int
```

Comment:
While I like sticking to paper notations, here I would go for more talkative variable names...